lib/__tests__/standalone.test.js

```diff
@@ -107,6 +107,81 @@ it("standalone with non-existent-file and allowEmptyInput false should throw err
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/non-existent-file.css`, `${fixturesPath}/another-non-existent-file.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with three or more non-existent-files and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/non-existent-file.css, ${fixturesPath}/another-non-existent-file.css and ${fixturesPath}/third-non-existent-file.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/non-existent-file.css`, `${fixturesPath}/another-non-existent-file.css`, `${fixturesPath}/third-non-existent-file.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with a glob that doesn't match anything and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/nodir/**/*.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [`${fixturesPath}/nodir/**/*.css`],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with two globs that don't match anything and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/nodir/**/*.css and ${fixturesPath}/anotherdir/**/*.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/nodir/**/*.css`, `${fixturesPath}/anotherdir/**/*.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
+
+it("standalone with three globs that don't match anything and allowEmptyInput false should throw error with code 80", () => {
+  const expectedError = new Error(`${fixturesPath}/nodir/**/*.css, ${fixturesPath}/anotherdir/**/*.css and ${fixturesPath}/athird/**/*.css does not match any files`)
+  expectedError.code = 80
+
+  return standalone({
+    files: [ `${fixturesPath}/nodir/**/*.css`, `${fixturesPath}/anotherdir/**/*.css`, `${fixturesPath}/athird/**/*.css` ],
+    config: configBlockNoEmpty,
+    allowEmptyInput: false,
+  }).then(() => {
+    throw new Error("should not have succeeded")
+  }).catch(actualError => {
+    expect(actualError).toEqual(expectedError)
+  })
+})
```

Comment:
The repeats are actually for files vs. globs (except for the first two tests on lines 81 and 95, which I didn't write). Maybe declaring `allowEmptyInput=false` is confusing though? Or are there just an excessive number of tests?